Solutions/PaloAlto-PAN-OS/Playbooks/PaloAltoCustomConnectorOnPrem/azuredeploy.json

```diff
@@ -0,0 +1,2478 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "metadata":{
+      "comments":"This PaloAlto custom connector uses PAN-OS API to perform different actions on the PAN-OS VM",
+      "author":"Sindhu Talluri,PAN-OS"
+   },
+   "parameters":{
+      "PaloAlto_PAN-OS_Rest_API_CustomConnector_Name":{
+         "defaultValue":"PAN-OSRestApiCustomConnector",
+         "type":"String",
+         "metadata":{
+            "description":"Name of the PaloAlto REST API Connector"
+         }
+      },
+      "PaloAlto_PAN-OS_XML_API_CustomConnector_Name":{
+         "defaultValue":"PAN-OSXmlApiCustomConnector",
```

Comment:
This title is already taken by other connector in the Solution. Please check once